src/components/posts/PostListItem.js

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import "./PostListItem.css"
+
+export const PostListItem = props => {
+  const { post } = props
+  const { title, user, category } = post
+
+  return (
+    <div className="PostListItem">
```

Comment:
You capitalized the first letter of all your classNames in this component, but in the imported CSS all of the classNames start with lower case... if you want to apply that CSS probably change all of the upper case here to lower case?

Reply:
We will fix this later.